### PR TITLE
Add installation instructions for MacPorts

### PR DESCRIPTION
**Pull request by gpanders**
##### Summary

Netdata was recently added to MacPorts! This PR adds installation instructions for MacPorts users.

##### Component Name

Website.

##### Test Plan

N/A

##### Additional Information

None

### CODE REVIEW

**Comment by joelhans** [comment]
Thanks for this! The docs LGTM me at first glance, but I'm pinging some extra people here in case they want to chime in with feedback or test out the process themselves. @OdysLam is very invested in the macOS installation story, so he'll want to check this out, and @Ferroin handles our packaging overall.

Lately we've been discussing how to handle official vs. community-driven installation methods in the documentation, so this PR will be a good forcing function for us to decide on a solution.

**Comment by OdysLam** [comment]
Hey @gpanders, 

Before we move forward, I want to test if the Agent illustrates 2 of the problems that exist with brew, namely:
1) Packaged with old dashboard (newer is downloaded from netdata/dashboard)
2) Doesn't work with Netdata Cloud. 

Your contribution is not only awesome but very timely since as @joelhans mentioned is a subject that we are discussing internally at the moment. Please bear with us as we try to chart a strategy. You rock eitherway ✌️

**Comment by gpanders** [comment]
> Before we move forward, I want to test if the Agent illustrates 2 of the problems that exist with brew, namely:
> 
>     1. Packaged with old dashboard (newer is downloaded from netdata/dashboard)
> 
>     2. Doesn't work with Netdata Cloud.

Yes I believe both of these are the case with MacPorts as well. Like brew, MacPorts is affected by #10842 which means we also are currently providing version 1.29.3. We are also unable to compile with Netdata Cloud support because the `configure` script [insists on using a static library for `libmosquitto`](https://github.com/netdata/netdata/blob/master/configure.ac#L653). MacPorts provides `libmosquitto` as a shared library but currently there is no mechanism to provide libraries statically.

We could work around this by compiling libmosquitto statically as one of the build steps in our Netdata port, but that is unconventional and we wouldn't go down that route unless 1) we knew for certain that Netdata *needs* libmosquitto to be statically compiled for some reason and 2) we had enough users requesting that feature to make it worth it.

**Comment by OdysLam** [comment]
@gpanders -- Thanks for the detailed response. You greatly saved me a bit of time from manually testing it myself. 

For the Cloud, we have released a version of the component that required `libmosquitto` that no longer requires it. While the new version can be built on Linux successfully, on MacOS we have a couple of bugs that we are root causing (cc @underhood ). At the moment we can't offer a timeframe.

For the dashboard, check our [FAQ](https://community.netdata.cloud/t/i-have-an-old-version-of-the-netdata-agent-dashboard/1207). You only need to download the latest release from netdata/dashboard and place the files into the correct directory. We are working on this front as well in order to make a long-term solution. cc @Ferroin for this. 

Again, we are humbled by your contribution and for taking the time to discuss these issues. Rest assured that they are top of mind, amongst a myriad other product and engineering-related matters.

**Comment by gpanders** [comment]
> For the dashboard, check our [FAQ](https://community.netdata.cloud/t/i-have-an-old-version-of-the-netdata-agent-dashboard/1207). You only need to download the latest release from netdata/dashboard and place the files into the correct directory. We are working on this front as well in order to make a long-term solution. cc @Ferroin for this.

Thanks for pointing this out -- I wasn't aware that the dashboard was now hosted separately. Our port should be updated [soon](https://github.com/macports/macports-ports/pull/10763).

**Comment by OdysLam** [comment]
Thanks @gpanders for improving your port. Rest assured that we are working towards solving both.

File: packaging/installer/methods/macos.md [hunk 1]
```diff
@@ -36,6 +37,25 @@ brew install netdata
...
+```
+
+You can have MacPorts automatically run the `netdata` service in the background and on restarts with:
+
+```bash
+sudo port load netdata
+```
+
+Your Netdata config file will be placed at `/opt/local/etc/netdata/netdata.conf`. You can find stock configuration
+files at `/opt/local/lib/netdata/conf.d/`.
```
Comment (ilyam8) [inline packaging/installer/methods/macos.md:55]: Is this path valid for m1 devices?

File: packaging/installer/methods/macos.md [hunk 2]
```diff
@@ -36,6 +37,25 @@ brew install netdata
 
 Skip on ahead to the [What's next?](#whats-next) section to find links to helpful post-installation guides.
 
+## Install Netdata with MacPorts
+
+To install Netdata with MacPorts, simply run:
```
Comment (ilyam8) [inline packaging/installer/methods/macos.md:42]: Let's add a link to the https://ports.macports.org/port/netdata/ where users can check the Agent version/additional info.

**Review by iigorkarpov — COMMENTED** [review]
I'd propose to make both descriptions similar:
```
-   [Install Netdata via the Homebrew package](#install-netdata-with-the-homebrew-package)
 -   [Install Netdata via MacPorts](#install-netdata-with-macports)
```
 Also, bash is not needed for Macports. Not sure about Homebrew though.

**Comment by ilyam8** [comment]
@iigorkarpov I'm not quite clear what you mean

- They are similar, see https://github.com/gpanders/netdata/blob/5dffa80ca115032bbdebce0ddc660a4f6b33c4da/packaging/installer/methods/macos.md?plain=1#L16-L17.
- I don't see "bash is required" anywhere.

File: packaging/installer/methods/macos.md (hunk 1 above)
Comment (gpanders) [reply packaging/installer/methods/macos.md:55]: Yes, MacPorts uses the same prefix on all devices.

**Comment by ilyam8** [comment]
Closing because [MacPorts Netdata](https://ports.macports.org/port/netdata/) is outdated (v 1.29.3). I think it is not an option for now.

**Pull request closed without merging**